SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -241,6 +241,9 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	UPROPERTY(EditAnywhere, Config, Category = "Replication", meta = (DisplayName = "Use RPC Ring Buffers"))
 	bool bUseRPCRingBuffers;
 
+	UPROPERTY(EditAnywhere, config, Category = "Threading")
```

Comment:
Can you add an `experimental` comment here

Reply:
Actually, I don't think we should expose it to the editor at all, just keep it as an advanced option

Reply:
Addressed.